manifests/vanilla/deploy-csi-snapshot-components.sh

```diff
@@ -50,15 +50,6 @@ if ! command -v kubectl > /dev/null; then
   exit 1
 fi
 
-feature_state=$(kubectl get configmap internal-feature-states.csi.vsphere.vmware.com -n vmware-system-csi -o jsonpath='{.data.block-volume-snapshot}')
-if [ "$feature_state" = "true" ]
-then
-        echo -e "✅ Verified that block-volume-snapshot feature is enabled"
-else
-        echo -e "❌ ERROR: Please enable the block-volume-snapshot feature to proceed"
-        exit 1
-fi
-
```

Comment:
Hi @deepakkinni , with vanilla v3.2 onwards, the idea is to not allow disabling already GA'ed features. Can you PTAL at this PR as well as I'm removing this check?

Reply:
This is fine, glad that you made the changes!